### DISCUSSION TITLE
[4.0][tinyMCE][Question] Parameter for expanding tinyMCE menu

### DISCUSSION THREAD

**Discussion by PhocaCz** [joomla/joomla-cms#36056] [General] [answered]
Hi,

I have a question regarding tinyMCE editor in Joomla 4. What is the name of parameter which makes the tinyMCE menu permanently opened?
![img](https://i.imgur.com/z51Hlvi.png)
Thank you, Jan

**Comment by brianteeman** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1296369/141865410-21406413-8ebe-4c29-baf6-274783802f7b.png)

**Reply by PhocaCz** [reply]
Setting Wrap did the trick. Thank you.
(It is even important to check the tinyMCE set)

**Comment by PhocaCz** [comment]
Thank you for the answer, this is what I have tested and still not opened as default:

![img](https://i.imgur.com/WvRf6rK.png)

![img](https://i.imgur.com/VxXYQQK.png)

![img](https://i.imgur.com/uVmy0KO.gif)

**Reply by brianteeman** [reply]
You have almost certainly come across an example of a poor user interface for the settings of tinymce.

TinyMCE lets you have different settings for different usergroups. The default group for super users is set 0. From the video it looks like you changed the value for set 4 which also looks like it has not been set up for any groups.

![image](https://user-images.githubusercontent.com/1296369/141867661-97a74a47-3b8f-4aa6-b4e2-6966ae2de15b.png)

**Reply by PhocaCz** [reply]
I started with Set 0 but didn't help, so tested with additional sets:

![img](https://i.imgur.com/oKMVaoJ.gif)

**Reply by PhocaCz** [reply]
But I have reset it now, sliding and floating have no effects, but wrap does the trick now. So thank you for your answers, for me, it is solved with wrap.

**Comment by brianteeman** [comment]
Sorry, looking again at my original answer it is confusing. I was pointing to the parameter  to change. Not which option would remove the ...

Am on my phone right now the one you want is wrap

**Reply by PhocaCz** [reply]
Ok, thank you anyway for the answer, it is solved.
